.github/dependabot.yml

```diff
@@ -20,10 +20,19 @@ updates:
       update-types:
         - "patch"
         - "minor"
+      exclude-patterns:
+          - "org.springframework.boot:*"
+          - "io.spring.dependency-management"
+          # All netty references are temporary overwrites that must be set carefully
+          # We do not need dependabot to send pull requests
```

Comment:
why not? once some lib bumps it to say our 4.1.125, should be good to merge, nah?

Reply:
Because at least for now, we are only overwriting it for Spring transitively. Upgrading to a non-compatible version will just cause problems. 

Even if 4.1.125.Final and 4.2.6.Final should be non breaking versions, experience says that Netty is very particular about conflicts and it just causes headaches. 

As a side note, this dependabot configuration was not fully correct and it still created the PR https://github.com/kafbat/kafka-ui/pull/1326 but I'd suggest closing it instead of merging it. 

I'll send a follow up to fix this

Reply:
@Haarolean Please see https://github.com/kafbat/kafka-ui/pull/1329